app/forms/supply_teacher_form.rb

```diff
@@ -0,0 +1,34 @@
+class SupplyTeacherForm < Form
+  attribute :employed_as_supply_teacher, :boolean
+
+  validates :employed_as_supply_teacher,
+    inclusion: {
+      in: [true, false],
+      message: ->(object, _) { object.i18n_errors_path("select_employed_as_supply_teacher") }
+    }
+
+  def initialize(claim:, journey:, params:)
+    super
+
+    self.employed_as_supply_teacher = permitted_params.fetch(
+      :employed_as_supply_teacher,
+      claim.eligibility.employed_as_supply_teacher
+    )
+  end
+
+  def save
+    return false unless valid?
+
+    update!({eligibility_attributes: {employed_as_supply_teacher:}})
+  end
+
+  private
+
+  def i18n_form_namespace
+    "supply_teacher"
+  end
```

Comment:
Suggestion: going with Rails' 'convention over configuration' principle perhaps this could be inferred from the class name by default.

Reply:
Update: See https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/2666/commits/64a3b62c3cc3be6dfd3619274be5d9807f5991e2#diff-b679b775ae8134204d714ca5ef2b0614a8c45bb674dfdb7c2467eedd69aa68caR47